website_v2/gcloud-powershell-website-v2/www/index.html

```diff
@@ -0,0 +1,70 @@
+﻿<!DOCTYPE html>
+<html class="google-js" lang="en" ng-app="powershellSite">
+<head>
+  <meta charset="utf-8">
+  <script>
+      (function(H){H.className=H.className.replace(/\bgoogle\b/,'google-js')})(document.documentElement)
```

Comment:
This is from the MAIA CSS framework. I also have no idea what this is used for, but we should probably keep it. (Maybe comment it to just say "MAIA CSS initialization."